src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Configuration/AppSettings.cs

```diff
@@ -75,6 +75,11 @@ public class AppSettings
         /// </summary>
         public string FormLayoutSettingsFileName { get; set; } = "Settings.json";
 
+        /// <summary>
+        /// Gets or sets the name of the layoutsets file name
+        /// </summary>
+        public string LayoutSetsFileName { get; set; } = "Layoutsets.json";
```

Comment:
@TheTechArch Should be lowercase, for example `layout-sets.json`. Don't use Settings.json as an example, ref. #5199.

Reply:
updated